idf_build_apps/app.py

```diff
@@ -95,6 +95,7 @@ class App(BaseModel):
 
     _build_log_filename: t.Optional[str] = None
     _size_json_filename: t.Optional[str] = None
+    _size_json_extra_args: t.Optional[t.List[str]] = None
```

Comment:
```suggestion
    size_json_extra_args: t.Optional[t.List[str]] = None
```

Attributes with a leading underscore (`_`) are private and used to support placeholder expansion.

This one doesn't support that feature, so it's better to use `size_json_extra_args` directly.

Reply:
Thanks. Fixed